Feat/complexes-page-on-react

n-e-t-t-u-n-o opened the pull request.

n-e-t-t-u-n-o committed:
complexes: cards added, images' paths  fixed

n-e-t-t-u-n-o committed:
fix: codeguide attributes order

imevro commented on src/App.js:
```diff
@@ -1,9 +1,10 @@
 import React, { Component } from "react";
-import "./App.css";
+import "./index.css";
```
поч не в соответствующем файле

imevro replied:
?

imevro commented on src/App.js:
```diff
@@ -12,8 +13,8 @@ class App extends Component {
-        <Discover />                 {/*
-        <Cards />         */}
+        <Discover />
+        <Cards />
```
читай чужие пры или спроси в чате у Говорухина почему это неправильно

imevro commented on src/Complexes/List/Card.jsx:
```diff
@@ -0,0 +1,69 @@
+export default (props) => {
+  return (
+    <Card to={`/Complexes/List/${props.id}`} href="" title="">
+      <CardImage src={props.imageSrc} alt={props.imageAlt} />
```
src={imageSrc}?

imevro commented on src/Complexes/List/Card.jsx:
```diff
@@ -0,0 +1,69 @@
+
+export default (props) => {
+  return (
+    <Card to={`/Complexes/List/${props.id}`} href="" title="">
```
что с ссылкой, почему она капитализирована

imevro replied:
почему to?

imevro replied:
`to` это проп реакт-роутеровского Линк, который ты не используешь (почему)

imevro commented on src/Complexes/List/Card.jsx:
```diff
@@ -0,0 +1,69 @@
+  return (
+    <Card to={`/Complexes/List/${props.id}`} href="" title="">
+      <CardImage src={props.imageSrc} alt={props.imageAlt} />
+      <CardDescription>
```
читай чужие пры про префиксы

imevro commented on src/Complexes/List/CardsList.jsx:
```diff
@@ -0,0 +1,42 @@
+import { Grid, Row, Col } from 'react-flexbox-grid';
+import Card from './Card'
+
+import CardImage1 from '../../img/bitmap1.jpg';
```
это компонент? почему с заглавной буквы тогда?

imevro commented on src/Complexes/List/CardsList.jsx:
```diff
@@ -0,0 +1,42 @@
+          imageAlt="764 Metropolitan Avenue"
+          area="South Beach, San Francisco"
+          title="764 Metropolitan Avenue"
+          text="Beautifully appointed residences perched atop Rincon Hill, The Harrison offers unparalleled Bay Bridge and city views within steps of San Francisco’s greatest restaurants, shops, and The Embarcadero." />
```
читай чужие пры

n-e-t-t-u-n-o committed:
react-router installed, index.jsx added

n-e-t-t-u-n-o committed:
fixed BodyClassName, fixed yarn

n-e-t-t-u-n-o committed:
fixed naming in components

n-e-t-t-u-n-o committed:
removed CardsList.jsx

n-e-t-t-u-n-o committed:
fixed link in Card.jsx

n-e-t-t-u-n-o committed:
react-helmet added

n-e-t-t-u-n-o committed:
App.css removed

imevro commented on package.json:
```diff
@@ -4,10 +4,14 @@
+    "react-router": "^4.1.1",
+    "react-router-dom": "^4.1.1",
+    "styled-components": "^2.1.0",
+    "yarn": "^0.24.5"
```
не понял

imevro commented on src/App.js:
```diff
@@ -1,22 +1,27 @@
+        <div className="CompassApp">
+          <Helmet>
+            <title>Compass Development</title>
+            <meta charset="UTF-8"/>
```
смысл

imevro commented on src/App.js:
```diff
@@ -1,22 +1,27 @@
-        <GlobalFooter />
-      </div>
+      <Router>
+        <div className="CompassApp">
```
классы у нас все еще snake-case

imevro commented on src/App.js:
```diff
@@ -1,22 +1,27 @@
+import {Helmet} from "react-helmet";
 import "./index.css";
 
 import GlobalHeader from './GlobalHeader'
```
можно грохнуть префикс глобал

imevro commented on src/Complexes/List/Card.jsx:
```diff
@@ -41,29 +44,29 @@ const CardArea = styled.p`
-        <CardTitle>{props.title}</CardTitle>
-        <CardText>{props.text}</CardText>
-      </CardDescription>
+    <Card to={`/complex/:id`} href="" title="">
```
не заработает

imevro replied:
из пропов бери

n-e-t-t-u-n-o committed:
fix: changed links to props.id
     removed meta and stuff from helmet
     changed className to snake-case
     removed global prefixes
